src/pages/organizers/[organizerId]/ownerships/index.page.tsx

```diff
@@ -90,19 +156,65 @@ const Ownership = () => {
                         variant={ButtonVariants.SUCCESS}
                         iconName={Icons.CHECK_CIRCLE}
                         spacing={3}
+                        onClick={() => {
+                          setIsQuestionModalVisible(true);
+                          setSelectedRequest(request);
+                          setActionType(ActionType.APPROVE);
+                        }}
                       >
                         {t('organizers.ownerships.table.actions.approve')}
                       </Button>
                       <Button
                         variant={ButtonVariants.DANGER}
                         iconName={Icons.TIMES_CIRCLE}
                         spacing={3}
+                        onClick={() => {
+                          setIsQuestionModalVisible(true);
+                          setSelectedRequest(request);
+                          setActionType(ActionType.REJECT);
+                        }}
                       >
                         {t('organizers.ownerships.table.actions.reject')}
                       </Button>
                     </Inline>
                   )}
                 />
+                <Modal
+                  title={
+                    isApproveAction
+                      ? t('organizers.ownerships.confirm_modal.title')
+                      : t('organizers.ownerships.reject_modal.title')
+                  }
+                  confirmTitle={
+                    isApproveAction
+                      ? t('organizers.ownerships.confirm_modal.confirm')
+                      : t('organizers.ownerships.reject_modal.confirm')
+                  }
+                  cancelTitle={
+                    isApproveAction
+                      ? t('organizers.ownerships.confirm_modal.cancel')
+                      : t('organizers.ownerships.reject_modal.cancel')
+                  }
```

Comment:
I would maybe prepare the path prefix in a variable and then use that everywhere else. Like 

```
const translationsPath = `organizers.ownerships.${isApproveAction ? 'confirm' : 'reject'}_modal`
```

Then you don't need all the ternaries everywhere